etc/jenkins/docker/image/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2022 Oracle and/or its affiliates. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Distribution License v. 1.0, which is available at
+# http://www.eclipse.org/org/documents/edl-v10.php.
+#
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
project is under `SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause`

Reply:
fixed.